src/Microsoft.DotNet.Darc/src/Darc/Operations/GatherDropOperation.cs

```diff
@@ -773,22 +820,22 @@ private bool IsBlobFeedUrl(string location)
         }
 
         /// <summary>
-        ///     Returns true if the location is a myget url.
+        ///     Determine whether this location is an azure devops build url.
         /// </summary>
         /// <param name="location">Location</param>
-        /// <returns>True if the location is a myget url, false otherwise.</returns>
+        /// <returns>True if the location is a sleet uri, false otherwise.</returns>
```

Comment:
```suggestion
        /// <returns>True if the location is an Azure Devops uri, false otherwise.</returns>
```

Is this what you meant? doesn't look like we're looking for sleet feeds.